PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json

```diff
@@ -227,6 +449,706 @@
+								"{{baseUrlClient}}"
+							],
+							"path": [
+								"articles"
+							],
+							"query": [
+								{
+									"key": "articleId",
+									"value": "{{articleId}}"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Create Article - Error - Invalid Category",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Category does not exist\", function () {\r",
+									"    pm.response.to.have.status(404);\r",
+									"    pm.expect(pm.response.json().message).to.eql(\"Category zzzyyyccc does not exist\");\r",
+									"});\r",
+									""
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									"var uuid = require('uuid');\r",
+									"var articleId = uuid.v4();\r",
+									"\r",
+									"pm.variables.set(\"articleId\", articleId);\r",
+									"pm.variables.set(\"length\", _.random(20,500));\r",
+									"pm.variables.set(\"width\", _.random(20,500));\r",
+									"pm.variables.set(\"height\", _.random(20,500));\r",
+									"pm.variables.set(\"weight\", _.random(100,1000));\r",
+									"pm.variables.set(\"piecesPerStorageBox\", _.random(1,500));\r",
+									"pm.variables.set(\"plannedSalesPerWeek\", _.random(1,200000));\r",
+									"pm.variables.set(\"minimumIncomingAmount\", _.random(1,1000));\r",
+									"pm.variables.set(\"category\", \"zzzyyyccc\");\r",
+									"\r",
+									"var statuses = [\"active\", \"inactive\"];\r",
+									"pm.variables.set(\"status\", statuses[Math.floor(Math.random() * statuses.length)]);"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "PUT",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n    \"length\": {{length}},\r\n    \"width\": {{width}},\r\n    \"height\": {{height}},\r\n    \"weight\": {{weight}},\r\n    \"piecesPerStorageBox\": {{piecesPerStorageBox}},\r\n    \"category\": \"{{category}}\",\r\n    \"dangerousGoods\": \"1\",\r\n    \"plannedSalesPerWeek\": {{plannedSalesPerWeek}},\r\n    \"minimumIncomingAmount\": {{minimumIncomingAmount}},\r\n    \"status\": \"{{status}}\"\r\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{baseUrlClient}}/articles?articleId={{articleId}}",
+							"host": [
+								"{{baseUrlClient}}"
+							],
+							"path": [
+								"articles"
+							],
+							"query": [
+								{
+									"key": "articleId",
+									"value": "{{articleId}}"
+								}
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Location",
+			"item": [
+				{
+					"name": "Create Location",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Stock was updated.\", function () {\r",
```

Comment:
The stock should imho not be mentioned here. May be location is correct?

Reply:
Yes, it should be location. Thanks